tfimm/train/train.py

```diff
@@ -80,9 +76,10 @@ def run(cfg: Union[ExperimentConfig, dict], parse_args: bool = True):
             resume=False,
         )
 
-    # In case we are using W&B sweep we need to adjust project name etc. to make them
-    # specific for each run in the sweep.
-    if cfg.sweep:
+    # In case we are using W&B sweep we need to adjust run name and ckpt dir
+    # To make the specific for each run in the sweep.
```

Comment:
Could you add a comment here to the effect of: When using sweeps, `wandb` sets the env variable `WANDB_RUNQUEUE_ITEM_ID`. The presence/absence of this variable allows us to detect if this run is part of a sweep.

And also add the more detailed comments that were previously part of the config class (L39-41) to explain why we are doing this.

Reply:
Never mind, I added the comments and merged the PR.